specification/purview/data-plane/Azure.Analytics.Purview.PDS/preview/2022-11-01-preview/pds.json

```diff
@@ -0,0 +1,463 @@
+            "schema": {
+              "$ref": "#/definitions/DatasourcePreferencesResponse"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DatasourcePreferences_Get": {
+            "$ref": "./examples/DatasourcePreferences_Get.json"
+          }
+        }
+      }
+    },
+    "/policy-elements": {
+      "get": {
+        "tags": [
+          "PolicyElements"
+        ],
+        "description": "Get policies for given scope.",
+        "operationId": "PolicyElements_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "scope",
+            "description": "Scope.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The API Version.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "filter",
+            "description": "filter.",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PolicyResponse"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "values"
+        },
+        "x-ms-examples": {
+          "PolicyElements_Get": {
+            "$ref": "./examples/PolicyElements_Get.json"
+          }
+        }
+      }
+    },
+    "/policy-events": {
+      "get": {
+        "tags": [
+          "PolicyEvents"
+        ],
+        "description": "Get policy events for given scope.",
+        "operationId": "PolicyEvents_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "scope",
+            "description": "Scope.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "syncToken",
+            "description": "The sync token.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PolicyEventsResponse"
+            }
+          },
+          "304": {
```

Comment:
The HTTP RFCs (specifically 7232) assign a very specific meaning to status code 304. It requires that a conditional header be sent and the condition is not met. That isn't the case here, so I'm not sure you should be using 304 in this non-standard way.

Reply:
Hi Mike, as discussed earlier, we are returning 304 to notify clients that there are no delta changes to return back.
Please check my explanation (in other conversation) on the sync token to corelate.
For this preview version we cannot change this value.
We will review this again for the GA version. Internally we are also discussing on the etags as you suggested.
But for now, I am resolving this conversation based on this explanation.